lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -151,8 +151,11 @@ static public function addGlobalIgnoredName($name)
      * Constructor.
      *
      * Initializes a new AnnotationReader.
+     *
+     * @param DocParser $parser
+     * @throws AnnotationException
```

Comment:
Ok, I've added this PHPDoc annotation because IDE told me that is incomplete. Fixed